backend/db/V2_1__change_user_column.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE brevity.user DROP COLUMN openID_id;
```

Comment:
why v2_1 and not just v3? should probably be consistent since we already have v2.

Reply:
yeah I forget to merge these when I know I can drop the schema and restart the migration